eclass/cargo.eclass

```diff
@@ -348,7 +351,7 @@ cargo_src_compile() {
 
 	tc-export AR CC CXX PKG_CONFIG
 
-	set -- cargo build $(usex debug "" --release) ${ECARGO_ARGS[@]} "$@"
+	set -- cargo build $(usex debug "" --release) --target "${RUST_TARGET}" ${ECARGO_ARGS[@]} "$@"
```

Comment:
please keep it as part of toml file, populated in `cargo_gen_config`
command line argument has highest priority and cargo bails out (usually) if same arg/param specified twice.
so passing it here prevents possibly passing it in ebuild without modifying the eclass.

refer to my patch https://gist.github.com/gyakovlev/f9bdcb59ceb3741f411ad23ebe0feed4


also note, specifying target for cargo changes output directory

`target/release` becomes `target/${RUST_TARGET}/release`

that's why I create stable symlink as target/gentoo

also it should handle USE=debug (that's not in my patch)

so stable symlink should be something like that

`target/${RUST_TARGET}/$(usex debug debug release)` `->` `target/gentoo`

and ebuilds just install files from `target/gentoo`

at least those ebuilds are affected, maybe more.
```
rg target/release --files-with-matches
gui-libs/greetd/greetd-0.6.1.ebuild
gui-libs/greetd/greetd-0.7.0.ebuild
net-libs/quiche/quiche-9999.ebuild
net-libs/quiche/quiche-0.8.1.ebuild
net-libs/quiche/quiche-0.7.0.ebuild
net-libs/quiche/quiche-0.8.0.ebuild
net-libs/quiche/quiche-0.5.1.ebuild
net-libs/quiche/quiche-0.6.0.ebuild
sys-apps/bat/bat-0.17.1-r1.ebuild
sys-apps/ripgrep/ripgrep-12.1.1.ebuild
dev-java/oracle-javamail/oracle-javamail-1.5.6-r1.ebuild
app-crypt/nitrocli/nitrocli-0.3.4.ebuild
app-crypt/nitrocli/nitrocli-0.4.0.ebuild
app-crypt/nitrocli/nitrocli-0.3.5.ebuild
```

Reply:
lost some parts of my comment...
I see you've updated `quiche`, but it still may require updating for USE=debug